runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -151,6 +152,8 @@ public void write(IToken token, Object data) throws OverwriteException {
                 // to validate the state of the write, we know that it didn't went through.
                 if (re instanceof OverwriteException) {
                     throw re;
+                } else if (re instanceof WriteTooLarge) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the usage of the "instanceof" operator by a catch block. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1193)